tests/e2e/config.ts

```diff
@@ -128,7 +128,7 @@ const config: Config = {
 			DB_PASSWORD: 'secret',
 			DB_PORT: '6100',
 			DB_DATABASE: 'directus',
-			PORT: '49152',
+			PORT: '59152',
```

Comment:
@licitdev Why is it needed ?

Reply:
These ports may already be in use by a root process: `remoted`, causing the tests to error out.
https://www.bleepingcomputer.com/forums/t/744822/question-regarding-remoted-its-purpose-and-link-local-addresses/